client/go/cmd/deploy.go

```diff
@@ -177,3 +180,17 @@ func waitForQueryService(sessionOrRunID int64) error {
 	}
 	return nil
 }
+
+func printPrepareLog(result vespa.PrepareResult) {
+	for _, entry := range result.LogLines {
+		switch l := entry.Level; l {
+		case "ERROR":
+			fmt.Fprint(os.Stderr, color.Red(l))
+		case "WARNING":
+			fmt.Fprint(os.Stderr, color.Yellow(l))
+		default:
+			fmt.Fprint(os.Stderr, color.Reset(l))
+		}
+		fmt.Fprintf(os.Stderr, " %s", entry.Message)
```

Comment:
All of these should print to `stderr`. Also, shouldn't there be a newline printed here? Or does `entry.Message` always contain one?

Reply:
Fixed.
And it looks like entry.Message always contains a newline, so we don't need to append that ourselves.